distribution_circuits_base/demo/demo.xml

```diff
@@ -71,7 +71,7 @@
             <field name="country_id" ref="base.be"/>
         </record>
 
-        <record id="partner_gac_1" model="res.partner">
+        <record id="distribution_circuits_base.partner_gac_1" model="res.partner">
```

Comment:
You're already in that module, why did you need to add the prefix? Why did you need to change this?

Reply:
in fact this was for solving an issue because the demo file was in data and not in demo. I just forget to remove this.